pkg/sqlcmd/errors.go

```diff
@@ -5,18 +5,19 @@ package sqlcmd
 
 import (
 	"errors"
-	"fmt"
 	"strings"
+
+	"github.com/microsoft/go-sqlcmd/internal/localizer"
 )
 
 // ErrorPrefix is the prefix for all sqlcmd-generated errors
-const ErrorPrefix = "Sqlcmd: Error: "
+var ErrorPrefix string = localizer.Sprintf("Sqlcmd: Error: ")
 
 // WarningPrefix is the prefix for all sqlcmd-generated warnings
-const WarningPrefix = "Sqlcmd: Warning: "
+var WarningPrefix string = localizer.Sprintf("Sqlcmd: Warning: ")
 
 // Common Sqlcmd error messages
-const ErrCmdDisabled = "ED and !!<command> commands, startup script, and environment variables are disabled"
+var ErrCmdDisabled string = localizer.Sprintf("ED and !!<command> commands, startup script, and enviroment variables are disabled")
```

Comment:
>enviroment [](http://example.com/codeflow?start=96&length=10)

environment

Reply:
Corrected it. I had copied error messages from ODBC sqlcmd.rll.lcl to our code to ensure that we have exact same error message and ended up copying the typo as well.